### PR TITLE
Update Eslint / Sass / Stylelint Dependencies

### PR DESCRIPTION
**Pull request by sghoweri**
## Jira
N/A

## Summary
Updates NPM dependencies for Eslint / Prettier, Sass / Sass Loader, Stylelint, and Nightwatch. 

No functional code changes other than auto-formatting  changes from Prettier + an update to the Webpack Sass loader config to use the new config syntax required. 

## How to test
Confirm build passes — updates for this should be super straightforward.

### CODE REVIEW

File: packages/build-tools/package.json [hunk 1]
```diff
@@ -82,14 +82,14 @@
     "prettier": "^1.17.1",
     "raw-loader": "^3.1.0",
     "resolve-url-loader": "^3.1.0",
-    "sass-loader": "^7.1.0",
     "sassdoc": "^2.7.0",
+    "sass-loader": "^8.0.0",
```
Comment (adamszalapski) [inline packages/build-tools/package.json]: We shouldn't add dependencies by hand and if we do that order them alphabetically.

File: packages/build-tools/package.json [hunk 2]
```diff
@@ -82,14 +82,14 @@
...
     "sassdoc-theme-default": "^2.8.2",
     "semver": "^6.1.1",
     "sharp": "^0.22.1",
     "string-replace-loader": "^2.1.1",
-    "style-loader": "^0.23.1",
     "svgo": "^1.2.2",
+    "style-loader": "^1.0.0",
```
Comment (adamszalapski) [inline packages/build-tools/package.json]: We shouldn't add dependencies by hand and if we do that order them alphabetically.

Comment (sghoweri) [reply packages/build-tools/package.json]: I think this was a goof — I'll get these deps reordered once this gets merged down!

Comment (sghoweri) [reply packages/build-tools/package.json]: 👍

**Comment by sghoweri** [comment]
PR was released with v2.13.0

**Pull request merged**